storage/__init__.py

```diff
@@ -9,24 +10,16 @@
 )
 from storage.fakestorage import FakeStorage
 from storage.distributedstorage import DistributedStorage
+from storage.storagecontext import StorageContext
+from storage.storagedriverclasses import STORAGE_DRIVER_CLASSES
 from storage.swift import SwiftStorage
 from storage.azurestorage import AzureStorage
 from storage.downloadproxy import DownloadProxy
-from util.ipresolver import NoopIPResolver
 
 TYPE_LOCAL_STORAGE = "LocalStorage"
 
-STORAGE_DRIVER_CLASSES = {
-    "LocalStorage": LocalStorage,
-    "S3Storage": S3Storage,
-    "GoogleCloudStorage": GoogleCloudStorage,
-    "RadosGWStorage": RadosGWStorage,
-    "SwiftStorage": SwiftStorage,
-    "CloudFrontedS3Storage": CloudFrontedS3Storage,
-    "AzureStorage": AzureStorage,
-    "RHOCSStorage": RHOCSStorage,
-    "CloudFlareStorage": CloudFlareS3Storage,
-}
+# Adding multi-CDN storage provider here to avoid circular import
+STORAGE_DRIVER_CLASSES["MultiCDNStorage"] = MultiCDNStorage
```

Comment:
Should be able to add it to the set, and avoid the circular import since you don't really need that in your multicdn file. See my other comments.

Reply:
fixed